serdaSimus/quiz/q1.js

```diff
@@ -0,0 +1,43 @@
+/*
+Problem Statement:
+
+Given an array of positive numbers and a positive number ‘k,’ find the maximum sum of any contiguous subarray of size ‘k’.
+
+Overview:
+Input: an array [2,3,2,5,5]
+output: 12
+
+This problem can be solved using window:
+    loop to visit all item in the array:
+        have a windowSize
+        as long as windowsize does not exceed K
+            calculate the sum by adding the next value
```

Comment:
This high-level overview leaves quite a bit to be desired. It should be a mix between what type of code you will write and a short story. Example below

-I'll start by init a windowStart set to 0
-init maxSum and a runningSum so that I can keep track of these through the program
-iterate over the array until I am at my desired windowSize
-now I can compare max against any running sum and leverage Math.max to update the value for me
-I will also remove my window start from running sum & increment my windowstart, effectively shrinking the 
    window from the left before the next iteration expands it from the right.
-once the loop is over I will return my maxSum